ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -966,6 +967,9 @@ TFLiteCore::TFLiteCore (const GstTensorFilterProperties *prop)
       interpreter = new TFLiteInterpreter ();
   } else
     interpreter = new TFLiteInterpreter ();
+
+  if (interpreter == NULL)
```

Comment:
Another comparing NULL with result of `new` here?

Reply:
This condition is to check if interpreter is not set when `checkSharedInterpreter (prop)` is true (line 966)!
If interpreter is `NULL` after `new TfliteCore()`, init function will create a seg fault so I added this compare logic.

Reply:
Oh that's right. Thanks!